### PR TITLE
Zipkin AWS Docker CI config

### PR DESCRIPTION
**Pull request by llinder**
This is a working Docker build using the Spring Boot module pattern.  Marking as WIP since this still needs CI config (circle?).

/cc @adriancole

### CODE REVIEW

File: zipkin-aws/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+ENV ZIPKIN_AWS_REPO https://jcenter.bintray.com
+ENV ZIPKIN_AWS_VERSION 0.1.0
+
+# Use to set heap, trust store or other system properties.
+ENV JAVA_OPTS -Djava.security.egd=file:/dev/./urandom
+
+WORKDIR /zipkin
+
+RUN apk add unzip && \
```
Comment (llinder) [inline zipkin-aws/Dockerfile]: need a different version of `unzip` due to a bug in the package provided in the base image.

File: zipkin-aws/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+FROM openzipkin/zipkin:1.20.0
+MAINTAINER OpenZipkin "http://zipkin.io/"
+
+ENV ZIPKIN_AWS_REPO https://jcenter.bintray.com
+ENV ZIPKIN_AWS_VERSION 0.1.0
+
+RUN apk add unzip && \
```
Comment (llinder) [inline zipkin-aws/Dockerfile]: Need a different version of `unzip` due to a bug in the package provided on the image.  The error I encountered was `unzip: short read` when extracting the module jar.

**Comment by adriancole** [comment]
I agree! it is working

**Comment by adriancole** [comment]
probably needs a README once we verify travis build works

**Comment by adriancole** [comment]
I had to change this to match the same dockerfile as zipkin-dependencies (then fuss a bit)

care to try it out? (maybe update the readme with a quick-start)

**Comment by adriancole** [comment]
@llinder @devinsba @openzipkin/devops-tooling 

so the release is the same as docker-zipkin-dependencies, which is [3-step](https://github.com/openzipkin/docker-zipkin-dependencies/issues/1) although both steps are local.

1. update the Dockerfile to the version you want.
2. push that commit to master.
3. push a tag with that version.

ex. make a commit that sets version to 0.1.1 in the Dockerfile. push that commit to master. then push tag 0.1.1 on that commit.

Although the credentials below are invalid, the fact they are tried means it works!

```bash
docker run -e 'SQS_QUEUE_URL=https://sqs.us-east-1.amazonaws.com/123456789/my-zipkin-queue' \
 -e 'SQS_AWS_ACCESS_KEY_ID=XqgzeGF3tC7u' \
 -e 'SQS_AWS_SECRET_ACCESS_KEY="F75uEjHM7ykLzXDJMTHrQ5Jr' \
 -p 9411:9411 openzipkin/zipkin-aws
```

**Comment by llinder** [comment]
Nice!  Thanks for ironing out the kinks.  I'm exited to replace my custom build with this in the morning :)

**Comment by llinder** [comment]
Couldn't wait until morning.  Just rolled our Zipkin cluster over to this images and everything is working great.  Thanks again for all the help!

**Pull request merged**